src/main/service.js

```diff
@@ -484,9 +575,13 @@ module.exports = {
     viewAllUsers: viewAllUsers,
     deleteUser: deleteUser,
     getAllMetaQuestions: getAllMetaQuestions,
+    getMetaQuestionsForExam,
```

Comment:
keep the already implemented standard of `key: value` even when they are the same

Reply:
Cant agree that continue bad habits is the right thing to do it. but who cares...